lib-android/src/main/res/layout/lib_button_sign_in.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:background="@color/lib_sign_in_google"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:layout_height="50dp"
+    >
+
+  <ImageView
+      android:id="@+id/lib_button_sign_in_icon"
+      android:layout_width="48dp"
```

Comment:
I'm going to move most of the values to styles or attrs, so the SignInButton can be easily styled.